e2e-test/java/io/cdap/plugin/slt/stepsdesign/StepDefinitionSLT.java

```diff
@@ -0,0 +1,472 @@
+
+
+
+    static {
+        CDFDriver = SeleniumDriver.getDriver();
+        sltLocators = SeleniumHelper.getPropertiesLocators(SLTLocators.class);
+
+        Properties prop = new Properties();
+        try {
+            prop.load(new FileInputStream("src/e2e-test/resources/Google_SAP_Connection.properties"));
+        } catch (IOException e) {
+            logger.error("Failed while reading SAP properties file" + e);
+        }
+        for (String property : prop.stringPropertyNames()) {
+            connection.put(property, prop.getProperty(property));
+        }
+    }
+
+    @Then("^Set Dummy MTID")
+    public void setDummyMTID() {
+        mtId = "02K";
+    }
+
+
+    @Given("^Open CDF replication and initiate pipeline creation")
+    public void open_cdf_replication() throws IOException, InterruptedException {
+        openCdf();
+        CDFDriver.get("http://localhost:11011/cdap/ns/default/replication/create");
+        sltActions.name_fill(UUID.randomUUID().toString().replaceAll("-", ""));
+        sltActions.next_click();
+        isErrorPresent = false;
+        bqRecordCount = 0;
+        sapRecordsCount = 0;
+    }
+
+    @When("^Source is SAP SLT fill connection parameters$")
+    public void source_is_sap_slt() throws IOException, InterruptedException {
+        sltActions.sltplugin_click();
+        sltActions.required_config(
+                "gcpProjectId",
+                "gcsDataPath",
+                mtId,
+                "sapJcoLibGcsPath",
+                "jcoclienthost",
+                "jcoclient",
+                "jcoclientsysnr",
+                "jcoclientuser",
+                "jcoclientpasswd");
+    }
+
+    @When("^Source is SAP SLT$")
+    public void source_is_sapslt() throws IOException, InterruptedException {
+        sltActions.sltplugin_click();
+    }
+
+    @Then("^User is able to set SLT parameter (.+) as (.+) and getting row (.+) for wrong input$")
+    public void userIsAbleToSetSLTParameterAsAndGettingRowForWrongInput(
+            String inputParameter, String inputValue, String errorMessage) {
+        errorExist = false;
+        elementHelper.replaceElementValue(sltLocators.inputParameter(inputParameter), inputValue);
+        sltActions.next_click();
+        errorExist = CDAPUtils.getErrorProp(errorMessage).
+                toLowerCase().contains(SLTLocators.rowError.getText().toLowerCase());
+        color = sltActions.rowError_color();
+        Assert.assertTrue(errorExist);
+        BeforeActions.scenario.write("Color of the text box" + color);
+        Assert.assertTrue(color.toLowerCase().contains("rgb(209, 86, 104)"));
+    }
+
+    @Then("^User is able to set SLT parameter \"([^\"]*)\" as \"([^\"]*)\"$")
+    public void userIsAbleToSetSLTParameter(
+            String inputParameter, String inputValue) {
+        elementHelper.replaceElementValue(sltLocators.inputParameter(inputParameter), inputValue);
+    }
+
+    @Then("^Replicate Existing Data is set to false$")
```

Comment:
Separate steps
1. Click/toggle  replicate existing data button/property
2. verify Replicate Existing Data is set to false